internal/bloblang/query/functions.go

```diff
@@ -675,7 +692,11 @@ func randomIntFunction(args *ParsedParams) (Function, error) {
 			r = rand.New(rand.NewSource(seed))
 		}
 
-		v := int64(r.Int())
+		n := max - min
+		if max < int64(math.MaxInt64) {
+			n++
+		}
```

Comment:
I'm sure there must be a simple explanation for this, but I'm struggling to get it... If `Int63n(n)` returns values from `[0,n)`, then I guess the goal is to obtain values in `[0,n]`, but please confirm and maybe leave a comment in the code if you don't mind :)

Reply:
Yes, the interval is half open so we need to increase the input parameter with 1 if we want to get random numbers equal to `max`. This check was to prevent it'd overflowing if max == maxInt64.
I've changed the doc and validation to make sure max is always smaller than maxInt64 so we can remove this check.

Reply:
This was a problem if we'd want to set min = max, when Int64n would panic with n = 0.
Maybe we want that max is always bigger than min?

Reply:
Even though I guess we'd expect a closed interval on `random_in` when specifying `max`, so we need to add 1 to `n` on `Int63n`